boards/raspberrypi/rpi_pico2/rpi_pico2_rp2350a_m33_w.dts

```diff
@@ -61,6 +75,7 @@
 		pinctrl-0 = <&pio0_spi0_default>;
 		pinctrl-names = "default";
 		status = "okay";
+		/*
 		airoc-wifi@0 {
 			compatible = "infineon,airoc-wifi";
```

Comment:
if this is wrong it should be removed in the original commit

Reply:
It is for the official wifi driver.